Gaijinizer patch/Map296.txt

```diff
@@ -17,10 +17,10 @@ I do not feel totally devastated ......#GT
 
 この世界のハーピーモドキは『繁殖期』だからな。
 > CONTEXT: Map296/events/55/pages/0/20/Dialogue/other1 
-Well, because "Ebony" is not dropped
-There may be that there is no huge hole ...
-This world's Harpy Beetles
-are in a "breeding season".#GT
+Well, without the "Black Coffin" being airdropped
+in, there's no massive hole here...
+But this world's Harpy Spawn are in their
+"Breeding Season", after all.#Amaboye #note
```

Comment:
from convos:
```
Well, without the "Black Coffin" being airdropped
in, there's no massive hole here... but it's
mainly because this world's Harpy Spawn are in
their "Breeding Season".
```